extendr-api/src/scalar/rbool.rs

```diff
@@ -3,6 +3,9 @@ use crate::scalar::Scalar;
 use crate::*;
 use std::convert::TryFrom;
 
+/// Integer (`i32`) values that [`Rbool`] is allowed to take.
+const VALID_VALUES: [i32; 3] = [0, 1, i32::MIN];
```

Comment:
Should we reference NA value here directly? NA value is an implementation detail & hardcoding it is a bit of an odd thing.

Reply:
Could you use `Rbool::na().inner()` or something like that to avoid hard coding?